README.md

```diff
@@ -6,10 +6,11 @@ Repository for Sensirion humidity and temperature sensor support on Arduino
 - SHTC3
 - SHTW1
 - SHTW2
+- SHT2x (SHT20, SHT21, SHT25)
 - SHT3x-DIS (I2C)
 - SHT3x-ARP (ratiometric analog voltage output)
-- SHT85
 - SHT4x
+- SHT85
```

Comment:
The SHT85 is SHT3x based, so should not be part of this commit

Reply:
Ah, ok. I just ordered it by number ... will revert this tomorrow.